<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChequeDeposit</class>
 <widget class="QDialog" name="ChequeDeposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cheque Deposit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Cheques to deposit:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="lstCheques">
     <column>
      <property name="text">
       <string>Customer Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Received</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1" colspan="2">
      <widget class="QTextEdit" name="txtComment"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QDateEdit" name="dtpDateDeposited">
       <property name="displayFormat">
        <string>dd MMM yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Bank:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date Deposited:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Comment:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="cboPrime">
       <item>
        <property name="text">
         <string>Prime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Guardian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Co-Op</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Giro</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>54</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdDeposit">
       <property name="text">
        <string>Deposit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>ChequeDeposit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
